### ISSUE TITLE
Mac m1 arm binary

### ISSUE THREAD

**Issue by jotka** [jenkins-x/jx#8067]
ho wiś it possible you don't have m1 arm binary in releases?

**Comment by tomhobson** [comment]
Hey, thanks for the issue. This is a good point and we should certainly do it. 

I'll have a look at the options for building the binaries

**Comment by luctrate** [comment]
Thanks, now there appears to be a darwin arm release. 
"jx-darwin-arm64.tar.gz"
And it works. But most plugins dont have one right now, which will lead to errors like 
jx health status
`ERROR: failed to install binary plugin jx-health version 0.0.75 to /Users/steven/.jx3/plugins/bin: unable to locate binary for arm64 darwin for health`

**Comment by paweljankowski** [comment]
Hey. Any progress?

**Comment by aabhas-sao** [comment]
Was investigating a little. So what I understood is we are using goreleaser to build and upload binaries with GitHub actions.
Although jx repository and jx-preview repository have similar .goreleaser.yml file and workflow files jx builds binaries for darwin-arm64 as well but jx-preview doesn't but both build linux-arm64.
Also I noticed that jx-health in jenkins-x-plugins org has no github actions file just .goreleaser.yml, so how are binaries pushed there automatically.
cc: @ankitm123 @tomhobson

**Comment by luctrate** [comment]
@aabhas-sao 
I suspect it's because of the hard-coded go version within the make file that the darwin arm is not built by the go releaser.
"GO_VERSION := $(shell $(GO) version | sed -e 's/^[^0-9.]([0-9.]).*/\1/')"
is from jenkins x cli make file, where it seems to work.
https://github.com/jenkins-x/jx/blob/main/Makefile

Go Version and darwin arm support.
https://github.com/goreleaser/goreleaser/issues/1952

I have created a PR to check. Pls have a look
https://github.com/jenkins-x-plugins/jx-preview/pull/354

**Comment by aabhas-sao** [comment]
@luctrate yeah that might help. Nice job noticing it. Let's hope it works.

**Comment by luctrate** [comment]
@aabhas-sao 
Sadly it did not fix the issue. 
Another try, the goreleaser version was set to 0.155.0 within the github actions file. 
For jx parent its 1.4.1
https://github.com/jenkins-x/jx/blob/main/.github/workflows/jenkins-x-release.yaml

Please check. 
https://github.com/jenkins-x-plugins/jx-preview/pull/356

**Comment by aabhas-sao** [comment]
> 

@luctrate Yes we also did find that. would open a PR to update goreleaser soon
.

**Comment by aabhas-sao** [comment]
@luctrate nice work. It worked. Now we can update other plugins as well.
